test/unit/core/type_traits/basic_test.cpp

```diff
@@ -34,3 +38,73 @@ TEST(type_trait, remove_cvref_t)
     EXPECT_FALSE((std::is_same_v<int*, seqan3::remove_cvref_t<int[3]>>));   // type stays same
     // the last example would be true for std::decay_t
 }
+
+//------------------------------------------------------------------------------
+// Tests for is_constexpr
+//------------------------------------------------------------------------------
+
+constexpr
+int    constexpr_nonvoid_free_fun(int i) { return i; }
```

Comment:
Since when are you in favour of aligning names with weird spaces? ^^
I would really like the `constexpr` on the same line as the function declaration since having them on top of the function is no in synch with our usual style (and it looks wierd).

Reply:
this is just copied over. I am not the author of this code. Anyway I see to fix this